data/brands/healthcare/laboratory.json

```
@@ -461,6 +461,18 @@
         "name:en": "Invitro",
         "name:ru": "Инвитро"
       }
```

Missing closing bracket
```suggestion
      }
},
```